### DISCUSSION TITLE
Centered wallpaper image with background color?

### DISCUSSION THREAD

**Discussion by justDeeevin** [nix-community/plasma-manager#414] [Q&A] [answered]
I use a wallpaper whose dimensions are too small to fill the screen. However, I want to preserve its size. In my current Plasma settings, I have it centered, and I can configure the rest of the background to be a solid color (the shade of black in the image). Can I accomplish this with plasma-manager? I tried using `wallpaperPlainColor`, but it can't be used in combination with a background image.

**Comment by HeitorAugustoLN** [comment]
Just implemented it in #429 and #427. #429 isn't merged yet, because CI is somehow failing.

**Comment by justDeeevin** [comment]
Doesn't seem to be working... am I using it wrong?
![image](https://github.com/user-attachments/assets/16489c02-89e9-4fa7-beaa-630ee660fb2c)
![image](https://github.com/user-attachments/assets/8823f8a3-57f7-4366-a526-afcacbe1693b)

**Comment by HeitorAugustoLN** [comment]
Support for the wallpaper `option` isn't there yet, the CI for the #429 is currently failing.

**Reply by HeitorAugustoLN** [reply, marked as answer]
But while #429 doesn't get merged, you can use the following input for plasma-manager `github:HeitorAugustoLN/plasma-manager/wallpaper-image-wallpaperbackground`, so it works.

**Reply by justDeeevin** [reply]
My apologies, I misunderstood what #427 did. Thanks for this workaround!

**Reply by HeitorAugustoLN** [reply]
#429 just got merged :)
